Source/Core/DolphinQt2/MainWindow.cpp

```diff
@@ -478,3 +493,105 @@ void MainWindow::SetStateSlot(int slot)
   Settings::Instance().SetStateSlot(slot);
   m_state_slot = slot;
 }
+
+void MainWindow::PerformOnlineUpdate(const std::string& region)
```

Comment:
Can you extract this to another file, since it's not part of the main window (aside from signal handling stuff)? Something like `WiiOnlineUpdate.(cpp|h)` would be fine.